CONTRIBUTING.md

```diff
@@ -144,104 +98,56 @@ into a single one that captures everything you're trying to do -- the history
+Resolves #<N>.
 
-We perform updates by pushing releases over-the-air to `~zod` approximately
-once per week, so any contribution that can be deployed OTA will usually find
-its way onto the network pretty rapidly.
+Thoroughly describe the changes made.
 
-If you want to propose a hotfix (i.e. a small, OTA-updateable change, usually a
-bugfix, to some currently-deployed release) then simply make it clear that your
-contribution -- whether it be a pull request, patch, or whatever -- is intended
-to be a hotfix.  A maintainer can then deploy it to the network outside of the
-normal release schedule.
-
-Less frequently we release new Vere versions, which requires users to download
-new binaries, and occasionally, while Urbit is still in early development, we
-breach network continuity in order to release large changes that are difficult
-to push out over-the-air.  Contributions to Vere, or non-OTA-able updates to
-Arvo, will find their way into releases before terribly long.
-
-## Code style
+### Related
 
-The Urbit project uses two-space indentation and avoids tab characters.
-In C code, it should not be too difficult to mimic the style of the code
-around you, which is just fairly standard K&R with braces on every
-compound statement. One thing to watch out for is top-level sections in
-source files that are denoted by comments and are actually indented one
-level.
-
-Hoon will be a less familiar language to many contributors.  We've published
-some [style guidelines for Hoon][hoon], but above all you should try to mimic
-the style of the code around you.  With regards to the style used throughout
-the codebase: the more recently the code was written, the more standard and
-accepted its style is likely to be.
-
-## Kernel development
+Reference any related issues, links, papers, etc. here.
+```
 
-Working on either C or non-kernel Hoon should not bring any surprises, but the
-Hoon kernel (anything under [`pkg/arvo/sys/`][sys]) is bootstrapped from a
-so-called *pill*, and must be recompiled if any changes are made. This should
-happen automatically when you make changes, but if it doesn't, the command to
-manually recompile and install the new kernel is `|reset` in `dojo`.  This
-rebuilds from the `sys` directory in the `base` desk in `%clay`.
+Tests will run automatically via GitHub Actions when you open a pull request or
+push new commits to an existing pull request.
 
-Currently, `|reset` does not reload apps like `dojo` itself, which will still
-reference the old kernel. To force them to reload, make a trivial edit to their
-main source file (under the `app` directory) in `%clay`.
+Once you've collected and addressed feedback, tests are passing, and your PR has
+been approved, merge the pull request. If you properly included the "Resolves
+#N." directive in the pull request description, merging will automatically close
+the tracking issue associated with the pull request.
```

Comment:
Doesn't this only happen once it (eventually) gets merged into master?

Reply:
As part of this I'd set the default branch to develop - I think then it closes as expected

Reply:
That feels like it might be closing too eagerly though. Mainnet people would still be experiencing the issue, naively search for it and find nothing, even though it's hidden away in closed issues. Maybe not a big deal, but worth taking into account.

Reply:
I think it's a setting, so we don't need it to automatically close the issue.